<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-family: Poppins;
	font-size: 16px;
}

QWidget {
	background-color: white;
}

QPushButton {
	border: none;
	background-color: white;
	font-size: 14px;
	font-family: Poppins;
	font-weight: 500;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>240</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
	color: #266676;
	font-size: 16px;
	border: 1px solid #266676;
}</string>
    </property>
    <property name="text">
     <string>Atr√°s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>240</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1a759f;
	color: white;
	font-size: 16px;
	font-weight: 500;
	height: 50px;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Siguiente</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetKeyboard" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>480</width>
      <height>135</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPlace">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
background-color: rgb(234, 234, 234);
font: 14px &quot;Poppins&quot;;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>Ej: Colegio de Manaure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nombre de la muestra&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>widgetKeyboard</zorder>
   <zorder>backBtn</zorder>
   <zorder>nextBtn</zorder>
   <zorder>inputPlace</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
